### PR TITLE
Move many things to env vars

### PR DESCRIPTION
**Pull request by jabrah**
* Feedback on reasonability of implementation, var names, etc

### CODE REVIEW

**Review by jaredgalanis — APPROVED** [review]
LGTM 👍

File: config/routes.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 const ensureAuthenticated = require('../middleware/ensure-auth');
 
+const env = process.env.NODE_ENV || 'development';
+const config = require('./config')[env];
```
Comment (jaredgalanis) [inline config/routes.js]: i think i originally passed config into the call to routes, but i don't think it matters

Comment (jabrah) [reply config/routes.js]: Would it be better to add the `config` object as a function param in the routes? would avoid this extra require statement and NODE_ENV check

Comment (jaredgalanis) [reply config/routes.js]: yes it would.

Comment (jaredgalanis) [reply config/routes.js]: small change so we might as well do it.

**Pull request merged**